tests/validation/tests/v3_api/resource/terraform/gke/gke.tf

```diff
@@ -0,0 +1,48 @@
+provider "google" {
+  region = var.region
+  version = "~> 3.42.0"
+  credentials = var.credentials
+  project = "rancher-qa"
+}
+
+module "gke_auth" {
+  source           = "terraform-google-modules/kubernetes-engine/google//modules/auth"
+
+  project_id       = "rancher-qa"
+  cluster_name     = var.cluster_name
+  location         = var.location
+  depends_on       = [google_container_cluster.default]
+}
+
+data "google_client_config" "current" {}
+
+data "google_container_engine_versions" "default" {
+  location = var.location
+}
+
+resource "google_container_cluster" "default" {
+  name               = var.cluster_name
+  location           = var.location
+  initial_node_count = 3
+  min_master_version = var.kubernetes_version
+  node_config {
+    machine_type = var.machine_type
+    disk_size_gb = var.disk_size
+  }
+}
+
+output "cluster_name" {
+  value = google_container_cluster.default.name
+}
+
+output "cluster_region" {
+  value = var.region
+}
+
+output "cluster_location" {
+  value = google_container_cluster.default.location
+}
```

Comment:
Why do we need to output those variables? I noticed that they are all defined in the variables.

Reply:
We can use these terraform configurations independently from the Rancher framework. So I decided to expose all the variables set in case that's needed.